source/opsmanual/screens.html.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Monitoring screens
+parent: /opsmanual.html
+layout: opsmanual_layout
+---
+
+# Monitoring screens
+
+Most teams in GOV.UK have screens set up to show data about pull requests and
+releases.
+
+Most often displayed are the [deploy lag radiator][deploy-lag] and the [fourth wall](https://github.com/alphagov/fourth-wall).
+
+## Search screen
+
+![Screen shot of the search screen](images/search-screen.png)
+
+The [search screen][search-screen] displays live data from GOV.UK. Includes number of people on GOV.UK, latest searches, trending and recent content. It's publicly accessible because there's sometimes personal data in the latest searches.
```

Comment:
Is there meant to be a negation in here somewhere? ("It's publicly accessible because")